examples/docking-protein-DNA/docking-protein-DNA-cmrest-test.cfg

```diff
@@ -0,0 +1,71 @@
+# ====================================================================
+# protein-DNA docking example
+
+# directory in which the scoring will be done
```

Comment:
```suggestion
# TODO: Add this as an integration test
# directory in which the scoring will be done
```

Reply:
Ok - but it was needed to actually allow to check the CNS outputs for error messages. It is like all other tests in the examples dir.  Do we have already integration tests for all other test cfg files?

For integration tests we can have much simpler workflows that one check one specific module.